app/models/schemas/doi/name_identifier.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "NameIdentifier",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "properties": {
+    "schemeUri": {
+      "type": ["string", "null"]
+    },
+    "nameIdentifier": {
+      "type": ["string", "null"]
+    },
+    "nameIdentifierScheme": {
+      "type": ["string", "null"]
+    }
+  },
+  "dependentRequired": {
+    "affiliationIdentifier": ["nameIdentifierScheme"]
```

Comment:
@svogt0511 Should this line be as follows?

`"nameIdentifier": ["nameIdentifierScheme"]`

As opposed to `"affiliationIdentifier": ["nameIdentifierScheme"]`?

Reply:
nameIdentifier also cannot be null since it has `nonemptycontentStringType`.